release/src/github.ts

```diff
@@ -192,3 +192,94 @@ export async function getIssueWithCache ({
 
   return issue?.data as Issue | null;
 }
+
+function checksPassed(checks: GithubCheck[]) {
+  const MIN_TOTAL_CHECKS = 95;
```

Comment:
but why?

Reply:
Magic!

v49 has this many checks. I'll make this clearer.